contracts/BaseStrategy.sol

```diff
@@ -317,16 +310,14 @@ abstract contract BaseStrategy {
         outstanding = vault.report(want.balanceOf(address(this)).sub(reserve));
     }
 
-    // Override this to add all tokens this contract manages on a *persistant* basis
-    // (e.g. not just for swapping back to want ephemerally)
-    // NOTE: Must inclide `want` token
-    function protectedTokens() internal virtual view returns (address[] memory) {
-        address[] memory protected = new address[](1);
-        protected[0] = address(want);
-        return protected;
-    }
+    // Override this to add all tokens/tokenized positions this contract manages
+    // on a *persistant* basis (e.g. not just for swapping back to want ephemerally)
+    // NOTE: Do *not* include `want`, already included in `sweep` below
+    function protectedTokens() internal virtual view returns (address[] memory);
```

Comment:
can you move it to the top with the other methods you need to override?

Reply:
:+1:

Reply:
well, actually I had been placing them near where they were being used, so that it's easier to see the context